js/renovation/ui/scroll_view/scrollable_simulated.tsx

```diff
@@ -344,18 +344,22 @@ export class ScrollableSimulated extends JSXComponent<ScrollableSimulatedPropsTy
     }
 
     this.updateHandler();
+    const isTopDefined = location.top !== undefined;
+    const isLeftDefined = location.left !== undefined;
```

Comment:
These "!== undefined" are not necessary because there is the code few line above:
```
    if (!location.top && !location.left) {
      return;
    }
```
How do you suggest to manage this case?

Reply:
One of these can be undefined after the "if"